### ISSUE TITLE
TupleSections for 2-tuples are suggested twice

### ISSUE THREAD

**Issue by googleson78** [ndmitchell/hlint#862]
```haskell
peon :: ~ » cat lol.hs
f = foo (\x -> (x, y))
peon :: ~ » hlint lol.hs
lol.hs:1:10: Suggestion: Use tuple-section
Found:
  \ x -> (x, y)
Perhaps:
  (, y)
Note: may require `{-# LANGUAGE TupleSections #-}` adding to the top of the file

lol.hs:1:10: Warning: Use tuple-section
Found:
  \ x -> (x, y)
Perhaps:
  (, y)
Note: may require `{-# LANGUAGE TupleSections #-}` adding to the top of the file

2 hints
peon :: ~ » hlint --version                                                                  1 ↵
HLint v2.2.9, (C) Neil Mitchell 2006-2020
```

I think this is due to the fact that there is both
* one "general" detection for `TupleSections` in `Lambda.hs` which works with tuples of any size
* a "specialised" version of this hint (for 2-tuples) is available in the default `data/hlint.yaml` file

I think the specialised version can be removed.

**Comment by ndmitchell** [comment]
Thanks, great bug report - indeed, I removed the specialised variant.
